src/main/resources/frontend/src/services/adminService.ts

```diff
@@ -66,9 +66,10 @@ export const userPatch = async (user: UserPatch)=> {
     }
 }
 
-export const submissionsLatestGet = async (): Promise<Submission[]> => {
+export const submissionsLatestGet = async (batchSize?: number): Promise<Submission[]> => {
```

Comment:
From a defensive perspective, I would argue that the default value behavior of this method should request a finite number of submissions. Then, a developer must specifically ask for all submissions in order to get them.

It's tricky because any value we put as the default will be arbitrary, and we don't have a good system in place to provide configurations. Ideally, we would change the paradigm of this method, but maybe it's not reasonable to do until we have other supporting systems in place.

Reply:
That seems to be the best way to do it from a defensive perspective. However, I'm not sure how to define a default amount without setting up the client-side-settings you mentioned in another comment